views/preferences.handlebars

```diff
@@ -0,0 +1,44 @@
+<div class="pb-3 mx-auto" style="width: 600px">
+    <div class="text-center" >
+        <h1>{{ title }}</h1>
+        <h6>Manage your account settings and get information on how to use Book Swap</h3><br>
+   </div>
+
+
+    <h3>Account Settings</h3>
+
+    <a href="/updateaccount">
+        <div class="card">
+            <div class="card-body">
+                Update Account Information
+            </div>
+        </div>
+    </a>
+    <a href="/resetpassword">
+        <div class="card">
+            <div class="card-body">
+                Reset Password
+            </div>
+        </div>
+    </a><br>
+
+    <h3>Information</h3>
+
+    <a href="/gettingstarted">
+        <div class="card">
+            <div class="card-body">
+                Getting Started With BookSwap
+            </div>
+        </div>
+    </a>
+    <a href="/faq">
+        <div class="card">
+            <div class="card-body">
+                Frequently Asked Questions
+            </div>
+        </div>
+    </a><br>
+
+<a href="/logout" class="btn btn-secondary btn-sm active" role="button">Log out</a>
```

Comment:
Is this just a page of links? I'm thinking since we already have most of these links elsewhere we should not have them on this page too, it seems redundant and confusing especially since they have to click the dropdown which contains these links to get here. I'd just leave Reset Password and Logout in the dropdown. I think we should maybe add an option to the main navbar called 'Help' that is a dropdown in the navbar (similar to the user one) that has the FAQ and Getting Started pages. I'd add a link to update their profile to the My Profile page so they can just go there and then if they want to update click 'Update My Profile' or something like that. Let me know if you have questions!

Reply:
I've made this change in the preferences page, profile page, and navbar. My only worry now is that the preferences page seems bare. All it contains is a link to update their profile. do you think preferences should go right to the update account settings page? We can rename it in the dropdown

Reply:
Yes, I would just eliminate the preferences page and I would just have a link on the My Profile page to go to update your account settings, that will be fairly intuitive to the user and eliminates us having to have a ton of options in the dropdown.